src/atwho.js

```diff
@@ -17,20 +17,21 @@ define([
 
     // stripped down version of http://codepen.io/ImagineProgramming/storydump/javascript-memoization-timeout
     var timed = function timed(f, timeout) {
-        var cache, time = 0;
+        var cache = {}, time = 0;
```

Comment:
Hmm, maybe I'm all wrong here. It looks like there is a cache created per atwho-enabled input on the page? If that's true, then it looks like it's local to the input so not leaking to other instances.